.github/pull_request_template.md

```diff
@@ -1,17 +1,9 @@
-# Why was this change made? 🤔
+# Why was this change made?
 
+<!-- Feel free to use whatever GitHub copy/pastes into the PR description above, or even just `Fixes #1234` if the PR is closely tied to an issue. -->
 
 
-# How was this change tested? 🤨
-
-⚡ ⚠ If this change has cross service impact (including writing to shared file systems or interacting with other SDR APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡
-
-⚡ ⚠ If this change updates the Argo UI, run all integration tests that use Argo and create a PR on the integration test repo to fix anything this change breaks. ⚡
-
-
-# Does your change introduce accessibility violations? 🩺
-
-⚡ ⚠ Please ensure this change does not introduce accessibility violations (at the WCAG A or AA conformance levels); if it does, include a rationale. See the [Infrastructure accessibility guide](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/infra-accessibility.md) for more detail. ⚡
-
```

Comment:
i'd be a little hesitant to totally remove any mention of accessibility, but maybe this could be combined with "How was this change tested?"

e.g. as a second comment (after "If this change has cross-service impact ..."):

```
<!-- If this change introduces accessibility violations as the WCAG A or AA conformance levels, please include a rationale.  Violations can be detected by tools such as the [Siteimprove Accessibility Checker browser extension](https://www.siteimprove.com/integrations/browser-extensions).  See the [Infrastructure accessibility guide](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/infra-accessibility.md). -->
```

Reply:
Added!